backend/internal/certificate.js

```diff
@@ -30,6 +30,7 @@ const internalCertificate = {
 	intervalTimeout:    1000 * 60 * 60, // 1 hour
 	interval:           null,
 	intervalProcessing: false,
+	renewBeforeExpirationBy: [7, 'days'],
```

Comment:
According to the certbot docs, `renew` will find all certificates that expire within 30 days. Maybe I should change this to 30 days?

Reply:
I would prefer 30 days.